### ISSUE TITLE
Reference particular instances

### ISSUE THREAD

**Issue by m1is** [echa/ng-scrollable#68]
I don't see a way to reference a particular instance of a scroller in a scope. I'd love to use this but it's beyond frustrating that there doesn't appear to be a way to do this. I'd also like to be able to programmatically scroll a particular instance to the bottom but when I send a broadcast it scrolls all instances to the bottom.

**Comment by echa** [comment]
Yes, you are right. I had prepared for that using the `id` property in the config struct but never got to implement event filtering based on id's. 

As of now when you set `<div ng-scrollable="{id=<int|string>}"></div>` the id will be used as argument in events sent by ng-scrollable. The other way around (filtering events sent by you based on id) is missing.
